### ISSUE TITLE
Mix task doesn't propagate the failure

### ISSUE THREAD

**Issue by edgurgel** [asaaki/mix-edip#3]
Hey I noticed that when the `make` call fails the mix task still says that the image is ready.

```
/bin/sh: docker: not found
package/make-image.mk:18: recipe for target 'build' failed
make[1]: *** [build] Error 127
make: *** [image] Error 2
make[1]: Leaving directory '/build'
package/Makefile:24: recipe for target 'image' failed
Makefile:51: recipe for target 'build-package' failed
make[1]: *** [build-package] Error 2
==> The image is ready! You can check with docker images
```

I saw that there's a message for this case but apparently it's not being checked if `do_cmd` got back an `:ok` or `error` tuple

https://github.com/asaaki/mix-edip/blob/master/lib/edip/utils.ex#L27

I can send a PR for this case!

**Comment by asaaki** [comment]
Hm, this is surely not intended behavior.

So, I'm looking forward to your PR!

Keep in mind: there is also a nice helper for aborting at [lib/edip/utils.ex#L17](https://github.com/asaaki/mix-edip/blob/master/lib/edip/utils.ex#L17).
So when we get back an error tuple, we should do something like this:

``` elixir
case do_cmd(...) do
  {:error, message} ->
    error(message)
    abort!
  _ ->
    :noop # Well, nothing failed, so nothing to do here. ;-)
end
```

Maybe you can even wrap this into it's own nice function (I'm bad at naming things, but something like `run_or_fail/2` should do its job).

**Comment by asaaki** [comment]
Closed by #4; release `v0.2.3` published.

Thank you for your help!
